src/components/password-strength-field/password-strength.ts

```diff
@@ -0,0 +1,142 @@
+import { Component, Input, SimpleChanges } from '@angular/core';
```

Comment:
This file (and others) use 2 spaces instead of 4. Please update to use 4 spaces (and double check if your editor is configured correctly).